sqlite3/lib/src/ffi/ffi.dart

```diff
@@ -69,6 +69,8 @@ extension ValueUtils on Pointer<sqlite3_value> {
   }
 }
 
+final utf8Encode = utf8.encoder.convert;
```

Comment:
I think we should only define this once, the usage in `memory.dart` can just import this one.